### DISCUSSION TITLE
client with ssd

### DISCUSSION THREAD

**Discussion by dyuman** [ltsp/ltsp#808] [Q&A] [answered]
We have a lab where all computers are LTSP clients, and now we are thinking of moving to a mixed setup where some computers will still be LTSP clients and a few will have a local SSD to boot from (for performance) but then users would still login using their accounts as on the LTSP server (and home folders mounted from the LTSP server). Essentially the same functionality of an "LTSP client" with the only difference that the OS is booted locally.

In the LTSP installation page, it is described to run the following command `apt install --install-recommends ltsp epoptes-client` when creating a chroot using the Virtual machine method, so I had the idea to run that command on the SSD computer which I want to use as client, but I suppose there must be some extra step required to tell the installation which LTSP server to connect to, etc.?

Would anyone know if something like that could work?

Thank you

**Comment by alkisg** [comment, marked as answer]
It's possible to write /srv/ltsp/images/x86_64.img into a client hard disk, and boot from that.
The easiest way to do that is to netboot the client normally, where the kernel and initramfs are received via the network from the LTSP server,
but then to specify `root=/dev/sda1 ltsp.image=/path/to/x86_64.img` in the ltsp.ipxe settings, so that the OS is loaded from the hard disk.

But note that currently, loading the OS needs just e.g. 150 MB,
but on the other hand Firefox keeps a 1 GB cache under /home/username!

I mean that the savings of what you're trying to do won't be as important as you think. In most cases, they're not worth the trouble. You'd save a lot more traffic if you had a local /home rather than a local rootfs.

So, my advice is "get an NVMe disk for /home for the server, and a second network card + bonding", to speed things up, and not bother with local rootfs at all.

Local /home makes sense when disk speed is an issue; local rootfs not so much.

**Comment by dyuman** [comment]
Sorry for the delay in answering, but yes, followed your advice of using NVMe as /home on the server and netbooting the clients as normal. Works well!
